packages/aws/src/s3.ts

```diff
@@ -53,12 +55,13 @@ export class S3BlobStore implements BlobStore {
 
   async putTemp(bytes: Uint8Array | stream.Readable): Promise<string> {
     const key = this.genKey()
+    // @NOTE abort results in error from aws-sdk "Upload aborted." with name "AbortError"
     const abortController = new AbortController()
     const timeout = setTimeout(
-      () => abortController.abort('upload timed out'),
```

Comment:
The aws-sdk was throwing the `reason` passed to `abort()` away, which was actually a little confusing, so I nixed the message/reason here.